docs/architecture/lb/nginx.md

```diff
@@ -0,0 +1,270 @@
+# NGINX
+
+NGINX is a popular solution, but there are important disadvantages to consider when using it for scaling syslog ingestion:
```

Comment:
If you choose NGINX as a solution, consider the following when using it to scaling syslog ingestion: